docs/pages/careers/react-engineer-docs-infra.md

```diff
@@ -0,0 +1,113 @@
+# React Engineer — Docs Infra
+
+<p class="description">You will drive the development of the documentation platform that powers all MUI products, enabling it to grow and establish itself as a mature product.</p>
+
+## Details of the role
+
+- **Location**: Remote (preference for UTC-5 to UTC+2).
+- **Type of work**: Full-time (contractor or employee [depending on circumstances](https://mui-org.notion.site/Hiring-FAQ-64763b756ae44c37b47b081f98915501#494af1f358794028beb4b7697b5d3102)).
+- **Level**: [IC3 or above](https://mui-org.notion.site/Leveling-at-MUI-5c30f9bfe65149d697f346447cef9db1).
+- We're a **remote** company, we prefer asynchronous communication over meetings.
+
+## The company
+
+MUI's story began in 2014 with Material UI, the most successful React implementation of Google's Material Design.
+Today, Material UI stands as one of the most popular open-source libraries on GitHub and has paved the way for the fully-fledged startup known as MUI (founded in 2019), which now boasts an ever-expanding ecosystem of React UI products.
+We're a company of 31+ people as of late 2023, and we're growing.
+
+## The products
+
+MUI is best known for our flagship product, Material UI—but this is just one of three core component libraries we maintain.
+Base UI is our headless component library, and Joy UI is a sister library to Material UI that implements our own in-house Joy Design system.
+We also host Design Kits and pre-built Templates.
+
+Beyond the core libraries, MUI X offers advanced components like the Data Grid, Date and Time Pickers, and Charts, for more complex user interactions and data visualization needs.
+
+We're also making ambitious moves to incorporate our full suite of components into Toolpad, a low-code admin builder tool for assembling full-stack apps faster than ever.
+
+Learn more about MUI's products in this blog post: [An introduction to the MUI ecosystem](https://mui.com/blog/mui-product-comparison/). (This is also the kind of content you might create!)
+
+## The culture
+
+MUI is a fully remote company with a team that spans the globe.
+The majority of our work is asynchronous, and we rely on written communication to collaborate.
+We're radically transparent: nearly all of our work happens in public.
+Each contributor has the freedom to decide how and when they work, and that work is primarily self-directed: it's your responsibility to define and complete your own tasks in a timely manner.
+
+Learn more about what we value as an organization: [Evolving MUI's core values and behaviors](https://mui.com/blog/2023-mui-values/).
+
+## Why we're hiring
```

Comment:
Two ideas on how to improve this:

- Give people a quick idea of what this is about, e.g. mentioning https://github.com/shuding/nextra as a comparable quick help people quickly understand what this role is about.
- Maybe share some of our wildest dream with what the docs-infra could support. Something that would make the person understand that it's not just about recreation a https://mintlify.com/ but more. Now in practice, what could we mention? Ideas: Support of saving custom theme for organizations, offline support, demo generation to toggle between Material Design and Joy Design, AI generation of customization for demos based on user inputs

Reply:
Sounds good! Rephrased things around to capture your feedback.